CHANGELOG.md

```diff
@@ -1327,17 +1327,17 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [dacaiguoguogmail](https://github.com/dacaiguoguogmail)
   [#8097](https://github.com/CocoaPods/CocoaPods/pull/8097)
 
-* Nests test specs host apps inside that Pod's directory for cleaner project 
+* Nests test specs host apps inside that Pod's directory for cleaner project
```

Comment:
this one actually needs two empty spaces in the end for markdown formatting to work.

Reply:
actually no, this is a multiline changelog